src/pages/Ingame/index.js

```diff
@@ -36,83 +45,99 @@ export default function Game() {
       <Footer>
         <li>현재 점수 : 100</li>
         <li>
-          <button type="button">다음 라운드로</button>
+          <button type="button">선택완료</button>
         </li>
       </Footer>
     </InGame>
   );
 }
 
 const InGame = styled.div`
+  height: 100%;
   padding: 40px;
-  background-color: whitesmoke;
+  background-color: #e0dede;
   text-align: center;
 `;
 
 const Header = styled.ul`
   display: flex;
   justify-content: space-between;
   align-items: baseline;
-  font-size: 36px;
-  font-weight: 500;
+  font-size: 24px;
+  font-weight: 700;
 
   li:first-child {
-    font-size: 24px;
+    font-size: 18px;
+  }
+
+  span {
+    margin: 3px;
+    cursor: pointer;
   }
 `;
 
 const Teams = styled.ul`
-  display: flex;
-  padding: 40px 0;
-  justify-content: center;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  padding: 20px 0;
 `;
 
 const Team = styled.ul`
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
-  flex-grow: 1;
-  height: 120px;
-  background-color: powderblue;
-  padding: 20px;
+  justify-content: space-around;
+  height: 80px;
+  background-color: #c1d0fb;
+  padding: 10px;
   margin: 10px;
   border-radius: 10px;
 `;
 
 const TeamName = styled.li`
-  font-size: 24px;
-  margin-bottom: 5px;
+  font-size: 18px;
 `;
 
 const SelectedCard = styled.li`
-  background-color: whitesmoke;
-  padding: 20px;
-  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto;
+  text-align: center;
+  .selected-card {
+    width: 30px;
+    height: 30px;
+    font-size: 12px;
+  }
+  .enemy-card {
+    width: 30px;
+    height: 30px;
+    font-size: 12px;
+    background-color: #e0dede;
+  }
 `;
 
 const Cards = styled.ul`
   display: flex;
   justify-content: center;
 `;
 const Card = styled.button`
```

Comment:
다들 한 줄 씩 띄어져 있던데 이 부분도 통일성을 위해 한 칸 줄 바꿈 해주시면 좋을 것 같습니다
```suggestion

const Card = styled.button`
```